### PR TITLE
Define zenodo doi

### PR DESCRIPTION
**Pull request by niekdejonge**
Define the zenodo doi at one location

### CODE REVIEW

**Review by eeko-kon — CHANGES_REQUESTED** [review]
Hi Niek - can you also add a zipped version of your zenodo link for the modules? 
So that we can simply "wget ..." and download only the necessary modules for building our own library?

**Comment by niekdejonge** [comment]
@eeko-kon 
Hi Efi, 
I will sent you an email about this, since I will soon merge this Pull request, which might make it hard to follow the conversation.
